assignments/objects.js

```diff
@@ -19,26 +20,75 @@ const example = {
 
 // Write your intern objects here:
 
+const Mitzi =  {
+  "id": 1,
+  "email": 'mmelloy0@psu.edu',
+  "first name": 'Mitzi',
+  "gender": 'F'
+}
+
+const Kennan = {
+  "id": 2,
+  "email": 'kdiben1@tinypick.com',
+  "first name": 'Kennan',
+  "gender": 'M'
+}
+
+const Keven = {
+  "id": 3,
+  "email": 'kmummery2@wikimedia.org',
+  "first name": 'Keven',
+  "gender": 'M',
+  "speak": function () {
+    console.log(`Hello, my name is ${Keven["first name"]}!`);
+  }
+}
+
+const Gannie = {
+  "id": 4,
+  "name": 'Gannie',
+  "email": 'gmartinson3@illinois.edu',
+  "gender": 'M'
+}
+
+const Antonietta = {
+  "id": 5,
+  "name": 'Antonietta',
+  "email": 'adaine5@samsung.com',
+  "gender": 'F',
+  "multiplyNums": function (x,y){
+    console.log(x*y);
+  }
+}
+
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
-// Mitzi's name
+// Mitzi's name 
+console.log(Mitzi.name);
 
 // Kennan's ID
+console.log(Kennan.id);
 
 // Keven's email
+console.log(Keven.email);
 
 // Gannie's name
+console.log(Gannie.name);
 
 // Antonietta's Gender
+console.log(Antonietta.gender);
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
+Keven.speak();
 
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
+Antonietta.multiplyNums(3,4);
```

Comment:
```suggestion
Antonietta.multiplyNums = (a, b) => a * b;
console.log(Antonietta.multiplyNums(3,4);
```
another way to do it using ES6 arrow syntax